services/proxy-api/libs/handler-lib.js

```diff
@@ -0,0 +1,32 @@
+import * as debug from "./debug-lib";
```

Comment:
A general comment on this file that pertains to the whole service. `proxy-api` needs added as a module to `deploy.sh` or it will not deploy. Currently the new copy of `proxy-api` you split out into it's own service is not being deployed to AWS.